README.md

```diff
@@ -34,7 +34,312 @@ This README documents the build, customisation and testing of these runtime imag
 
 **Do you want to learn more about using Node.js actions to build serverless applications?** Please see the main project documentation [here](https://github.com/apache/openwhisk/blob/master/docs/actions-nodejs.md) for that information.
 
-## Usage
+## Build Runtimes
+
+### You have 2 options to build the NodeJS runtime:
+- Building locally
+- Using OpenWhisk Actions.
```

Comment:
The options should be intra-document hyperlinks (to section titles for those sections)